packages/prisma-generator-proto/src/generate/hooks.ts

```diff
@@ -1,17 +1,62 @@
 import { DMMF } from "@prisma/generator-helper";
 
 function generateSingleHook(model: DMMF.Model): string {
+  const initialValue: { [key: string]: string | number } = {};
+  for (const field of model.fields) {
+    if (field.kind === "scalar" && field.name !== "id") {
+      initialValue[field.name] = "";
+    }
+  }
+
   return `
-export const use${model.name} = () => {
-  const [state, setState] = useState<${model.name}>();
-  return state;
+export const use${model.name} = (): [${
+    model.name
+  }[], React.FormEventHandler<HTMLFormElement>, Omit<${
+    model.name
+  }, "id">, React.Dispatch<React.SetStateAction<Omit<${
```

Comment:
Let's actually find out the name of ID field like I mentioned in my previous comment.